cunumeric/_ufunc/ufunc.py

```diff
@@ -186,6 +186,10 @@ def __init__(self, name: str, doc: str) -> None:
         self._name = name
         self.__doc__ = doc
 
+    @property
+    def __name__(self) -> str:
```

Comment:
Actually this isn't needed, but it's presumably part of the documented ufunc API, so it doesn't hurt (?)

Reply:
I would expect it
```

In [1]: import numpy as np

In [2]: np.add
Out[2]: <ufunc 'add'>

In [3]: np.add.__name__
Out[3]: 'add'
```